index.html

```diff
@@ -49,5 +49,167 @@ <h2>I'am a Sotfware Developer</h2>
           </div>
         </section>
     </main>
+    <div class="main-body">  
+      <section class="main-row">
```

Comment:
- [ ] Let's use `display: grid;` to layout this section, as this is part of the requirements from Microverse.

> #### Works
> 
> 1. The container of the section is a `section` tag
> 2. The main container of the projects has a `display:grid`